### DISCUSSION TITLE
Setup of Cosmo DB for the demo

### DISCUSSION THREAD

**Discussion by sudhansuroy1** [microsoft/sample-app-aoai-chatGPT#325] [General]
Hi,

I need detailed steps for setting up the cosmo db and creating objects inside the db to support the chat history of the demo app.
Can any one guide me please.

**Comment by agrecaBAS** [comment]
This might not be exactly what you're looking for but it might help you start to use it. We just used the "Deploy To" > "A new web app" option via the Azure Open AI Studio Chat Playground 
![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/79329756/eb7c1f4e-720d-42ad-a69f-4a19c4508185)

Then just make sure that "Enable chat history in the web app" is checked before deploying.
![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/79329756/e9bd6e31-12dc-4cf8-a7db-0d37f5d1cfc4)
